chapters/en/slides/module6/module6_02.Rmd

```diff
@@ -0,0 +1,352 @@
+    ]
+  }
+}
+```
+
+Notes:
+Up until now,
+all the data we have plotted in Altair have been read from CSV-file
+as tidy dataframes.
+To visualize data on maps we will be using
+file formats that are designed specifically for geographic data.
+
+These are called GeoJSON and TopoJSON
+and are saved as plain text file,
+but with rules for how store geographic data such as coordinates.
+You can see an example of how they look in this slide.
+
+---
+
+## Creating a map in Altair
+
+```{python}
+import altair as alt
+from vega_datasets import data
+
+world_map = alt.topo_feature(data.world_110m.url, 'countries')
+alt.Chart(world_map).mark_geoshape()
+```
+
+Notes:
+The vega sample data sets contain a TopoJSON dataset for the world and the US,
+so let's use that here.
+
+Altair work directly with GeoJSON files,
+so to use the TopoJSON sample data,
+we use the `alt.topo_feature` help function
+so that Altair can read the TopoJSON file correctly.
+
+We then use `mark_geoshape` to visualize this geographic TopoJSON data
+in the form of a map.
+Since the data contains the border coordinates of each country,
+we can see these drawn as white lines in the chart on this slide.
+
+But why does Antarctica look so much bigger
+than the rest of the world?
+
+---
+
+## Changing the projection gives a more accurate representation of areas
+
+```{python}
+alt.Chart(world_map).mark_geoshape().project(type='equalEarth')
+```
+
+Notes:
+Since the Earth is a sphere
+it is not entirely straightforward
+how to represent the areas of the different parts of the world
+on a 2D visualization.
+
+The reason Antarctica looked so giant in the previous slide
+is because of how the spheres is being stretched 
+when it is projected onto a 2D surface.
+
+The default projection method for maps is often `mercator`,
+but it causes heavy distortion of the relative areas of land masses,
+which get worse the closer to the poles we come.
+
+Fortunately,
+many people are moving away from using the projection
+because it gives an incorrect impression of how the world looks.
+A commonly used alternative is the `equalEarth` projection,
+which focuses on correctly representing the relative areas of all landmasses.
+
+Some of the most noticeable differences in this projection
+are that Africa appears much bigger
+and Greenland, Antarctica and Russia appear notably smaller.
+Africa is in fact 15x the area of Greenland,
+far from the `mercator` projection which shows them as roughly the same size.
+
+---
+
+## The fill and border colors of countries can be changed
+
+```{python}
+alt.Chart(world_map).mark_geoshape(
+    color='black', stroke='#706545', strokeWidth=1
+).project(type='equalEarth')
+```
+
+Notes:
+We can change the color of both the filled are and the border 
+for each country.
+Just like with other colors in Altair,
+we can use either a named color
+or and exact hex-code.
+
+Here we also make the stroke width slightly larger 
+so that it is easier to see
+the country borders.
```

Comment:
```suggestion
We can change the colour of both the filled maps areas and the 
geographical borders.

And similar to other colour mappings in Altair,
we can use either a named colour
or and exact hex-code.

Here we use the named colour black, with the `color` 
argument to fill the map areas black, and a hexcode
with the `stroke` argument to change the geographical borders
to a grey colour. We also set `strokeWidth` to 1, so that 
it is easier to see the country borders.
```